app/models/post.rb

```diff
@@ -1,3 +1,18 @@
 class Post < ApplicationRecord
```

Comment:
- [OPTIONAL] As suggested by the previous reviewer, please update the datatype for the `comments_counter` and the `likes_counter`.

Shot |
:------:|
![a1](https://user-images.githubusercontent.com/39523312/166506014-eb3ffd9d-a66d-4fe6-b67c-5a61eda06836.png)